docker/pg_tests/scripts/tests/pgbackrest_backup_fetch_test.sh

```diff
@@ -22,14 +22,18 @@ echo "archive_timeout = 600" >> ${PGDATA}/postgresql.conf
 mkdir -m 770 /tmp/pgbackrest-backups
 
 pgbackrest --stanza=main --pg1-path=${PGDATA} --repo1-path=/tmp/pgbackrest-backups stanza-create
-pgbench -i -s 5 postgres
 
+pgbench -i -s 5 postgres
+pg_dumpall -f /tmp/dump1
+pgbench -c 2 -T 100000000 -S &
+sleep 1
 pgbackrest --stanza=main --pg1-path=${PGDATA} --repo1-path=/tmp/pgbackrest-backups backup
 
 /usr/lib/postgresql/10/bin/pg_ctl -D ${PGDATA} -w stop
```

Comment:
Let's do an online backup just like it is done in the wal-g backup-fetch test since we support the `wal-fetch` now.